framework/include/media/MediaRecorder.h

```diff
@@ -202,6 +202,29 @@ class MediaRecorder
 	 */
 	recorder_result_t setDuration(int second);
 
+	/**
+	 * @brief Limit the size of the file to be recoreded.
+	 * This should be called after setDataSource but before prepare
+	 * @details @b #include <media/MediaRecorder.h>
+	 * This function is a synchronous api
+	 * It sets the user's function
+	 * @param[in] Max size(byte), No limitation If zero or negative.
+	 * @return The result of setting the size
+	 * @since TizenRT v2.0
+	 */
+	recorder_result_t setSizeLimit(int byte);
+
+	/**
+	 * @brief Set Unlimited recording, regardless of duration or size.
+	 * This should be called after setDataSource but before prepare
+	 * @details @b #include <media/MediaRecorder.h>
+	 * This function is a synchronous api
+	 * It sets the user's function.
+	 * @return The result of setting unlimited recording
+	 * @since TizenRT v2.0
+	 */
+	recorder_result_t setUnlimited();
```

Comment:
@Taejun-Kwon 
PR is updated. Review Please.

- Removed setUnlimited() / Added `else` statement to disable the limit(duration or file size).
- Did not change about `mTotalFrames` setting statement because of pcm config.